src/app/alert/alert.demo.module.ts

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
This comment is _not_ something I want you to change right now, but I'm writing it down before I forget.

I was thinking about it this morning, and I think we should make it clearer what these are. I would write them in a separate `src/sandbox/` folder, that would be used by `src/app`. Of course, we'd remove a lot of the demos to stick to a single fully-featured example most of the time. So we'd do this as part of the demo cleanup rather than this PR.